src/Form/DataTransformer/ModelToIdPropertyTransformer.php

```diff
@@ -167,29 +175,26 @@ public function transform($value): array
             }
         }
 
-        if ('' === $this->property) {
-            throw new \RuntimeException('Please define "property" parameter.');
-        }
-
         $labels = [];
 
         /** @phpstan-var array<T>|\Traversable<T> $collection */
         foreach ($collection as $model) {
-            $id = current($this->modelManager->getIdentifierValues($model));
```

Comment:
current of identifiervalues is the same than implode https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/4.x/src/Model/ModelManager.php#L293 for entity with only one identifier

Reply:
Do you mean that for multiple identifiers, this wasnt working previously?

Reply:
Yes